### PR TITLE
[VTP-2016] Fixing a bug in migration script.

### PR DESCRIPTION
**Pull request by kayakyakr**
Fixing a bug in migrations that wasn't allowing more than a single migration to be run at a time.

Threw in a source environment option.

### CODE REVIEW

**Review by mxilievski — APPROVED** [review]
lgtm

**Review by amukoski — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ Name | Type | Required | Default  | Description
 --- | --- | --- | --- | ---
 **space_id**             | `string`  | Yes | `undefined` | The id of the contentful space
 **management_api_key**   | `string`  | Yes | `undefined` | The management-api key for contentful
-actions                  | `string[]`| No  | `createEnvironment, createCDAToken, applyMigration, updateAlias, cleanUpEnvironments` | The actions to run on this workflow. Comma separated string
+actions                  | `string[]`| No  | `createEnvironment, createCDAToken, applyMigrations, updateAlias, cleanUpEnvironments` | The actions to run on this workflow. Comma separated string. One of `createEnvironment, backupEnvironemnt, createCDAToken, applyMigrations, updateAlias, cleanUpEnvironments`
```
Comment (christang0807) [inline README.md:99]: Minor issue but `backupEnvironment` should probably also be added to the description, so:
`backupEnvironment, createEnvironment, createCDAToken, applyMigrations, updateAlias, cleanUpEnvironments`
and `backupEnvironemnt` is misspelled 😅

**Review by christang0807 — APPROVED** [review]
Aside from the 1 minor comment, everything else looks good

File: README.md (hunk 1 above)
Comment (kayakyakr) [reply README.md:99]: The first part is the default, backup is not default. Thanks for catching the misspelling >_<

**Review by mxilievski — APPROVED** [review]

**Pull request merged**